examples/scaling-criteo/03-Training-with-HugeCTR.ipynb

```diff
@@ -78,27 +78,22 @@
    ]
```

Comment:
<p>written in C++ ('in' missing)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>